.changeset/metal-socks-add.md

```diff
@@ -0,0 +1,5 @@
+---
+"vscode-ui5-language-assistant": patch
```

Comment:
please always include `"@ui5-language-assistant/vscode-ui5-language-assistant-bas-ext"` this package in change set.